File: CODIGO 10/work.kt
fun main(args: Array<String>) {
    val stdin = System.`in`
    val n = stdin.readLine().toInt()
    val heights = mutableListOf<Int>()
    for (i in 0 until n) {
        val line = stdin.readLine().split(" ")
        val d1 = line[0].toInt()
        val a = line[1].toDouble()
        val b = line[2].toDouble()
        val h = tan(b) * d1 / (tan(a) - tan(b))
        heights.add(h.roundToInt())
    }
    heights.forEach { println(it) }
}

File: CODIGO 10/work.scala
object CloudAltitudeMeasurement {

  def main(args: Array[String]): Unit = {
    val n = scala.io.StdIn.readLine().toInt

    val heights = for (i <- 0 until n) yield {
      val d1, a, b = scala.io.StdIn.readLine().split(" ").map(_.toDouble)

      val h = tan(b) * d1 / (tan(a) - tan(b))

      Math.round(h).toInt
    }

    println(heights.mkString(" "))
  }
}
